TicTacToe.cpp

```diff
@@ -243,3 +243,27 @@ void TicTacToe::changeTurn(){
 	actionsLeft = 3;
 	teamSide = teamSide * (-1);
 }
+
+void TicTacToe::viewTop() {
+ 	   for (int y = 2; y >= 0; y--) {
+        for (int x = 0; x < 3; x++) {
+            std::deque<int>* square = &field[y][x];
+            int size = square->size();
+						int value = square->at(2); // will be 1 (blue) or -1 (red) or 0 (empty)
```

Comment:
Th ONLY issue I see is that `std::deque.at()` will throw an exception if the is no element at position 2. You should probably have an if statement to check that it exists first.